### PR TITLE
Editing mobile web chapter

### LINKED ISSUE #2151: Mobile Web 2021
# Part II Chapter 14: Mobile Web

Mobile Web illustration

If you're interested in contributing to the Mobile Web chapter of the 2021 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| --- | ------- | --------- | -------- | ------- | -- |
| @ashleyish | @ashleyish @fellowhuman1101 @dwsmart @rvth   | @fili @obto | @ashleyish @obto | @logicalphase | @obto |

<details>
<summary><b>Expand for more information about each role</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [**0. Form the content team**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
+ [x] **May 31**: The content team has at least one author, reviewer, and analyst

### [**1. Plan content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
+ [x] **June 15** The content team has completed the chapter outline in the [draft doc][~google-doc]

### [**2. Gather data**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
+ [x] **June 30**: Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress
- **July 1 - 31**: HTTP Archive runs the July crawl

### [**3. Validate results**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
+ [x] **September 30**: Analysts have queried all metrics and saved the output to the [results sheet][~google-sheets]

### [**4. Draft content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
+ [x] **October 31**: The content team has written, reviewed, and edited the chapter in the [doc][~google-doc]

### [**5. Publication**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
+ [x] **November 15**: The completed chapter and all required [metadata](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) and [figures](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) are converted to markdown and submitted to [GitHub][~chapter-markdown]
- **December 1**: Target launch date 🚀

## Chapter resources

Refer to these 2021 Mobile Web resources throughout the content creation process:

[📄 Google Docs][~google-doc] for outlining and drafting content
[🔍 SQL files][~sql-files] for committing the queries used during analysis
[📊 Google Sheets][~google-sheets] for saving the results of queries
[📝 Markdown file][~chapter-markdown] for publishing content and managing public metadata

[~google-doc]: https://docs.google.com/document/d/1wQVzYjfbQUs6wmG6ldIE9Rcq4062hBCktrw9EnT9p2g/edit?usp=sharing
[~sql-files]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2021/mobile-web
[~google-sheets]: https://docs.google.com/spreadsheets/d/1mdma245ja_THTBApaJTeS4vmLY_Fn8VC6Kd8qx7wp-o/edit?usp=sharing
[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2021/mobile-web.md

### PR DESCRIPTION
**Pull request by shantsis**
Progress on #2525
Closes #2151 

Stage link: https://20211119t010018-dot-webalmanac.uk.r.appspot.com/en/2021/mobile-web

@ashleyish you're also missing your bio. I tagged you in a couple spots, including most of the figure descriptions because [they don't describe the actual stats](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide#writing-good-figure-descriptions)

### CODE REVIEW

**Comment by shantsis** [comment]
@tunetheweb / @rviscomi this chapter has 2 GIFs for images which are great (they convey the message), but they also make me wonder if we need a mechanism to stop or pause them for accessibility. https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide

**Comment by rviscomi** [comment]
It'd be nice to support `prefers-reduced-motion` and use a still image instead. The first part of that is a task for developers and the second half would require asking the authors for a still version of the gif. WDYT?

**Comment by shantsis** [comment]
I was thinking that too. In this case I'm not sure how the static message would show the delay they're exampling, but it could still work.
Is it possible to play the gif once or twice and then pause it the way MS teams does?

File: src/content/en/2021/mobile-web.md [hunk 1]
```diff
@@ -5,29 +5,30 @@ description: Mobile Web chapter of the 2021 Web Almanac covering page web vitals
 authors: [fellowhuman1101, dwsmart, ashleyish]
 reviewers: [obto, fili]
 analysts: [rvth, obto]
-editors: []
+editors: [shantsis]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1mdma245ja_THTBApaJTeS4vmLY_Fn8VC6Kd8qx7wp-o/
 fellowhuman1101_bio: Jamie Indigo isn’t a robot, but speaks bot.  As a technical SEO consultant at <a href="https://www.deepcrawl.com">Deepcrawl</a>, they study how search engines crawl, render, and index the web.  They love to tame wild JavaScript frameworks and optimize rendering strategies. When not working, Jamie likes horror movies, graphic novels, and Dungeons & Dragons.
 dwsmart_bio: Dave Smart is a developer and technical search engine consultant at <a href="https://tamthebots.com">Tame the Bots</a>. They love building tools and experimenting with the modern web, and can often be found at the front in a gig or two.
+ashleyish_bio:
```
Comment (shantsis) [inline src/content/en/2021/mobile-web.md:13]: @ashleyish add your bio here :)

**Comment by tunetheweb** [comment]
For the GIF issue, there's interesting tips here: https://css-tricks.com/gifs-and-prefers-reduced-motion/

To demonstrate the motion without the gif, you can just have two images side by side ([like I did here](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/):

image

We'll probably need something like this for the ebook anyway.

**Comment by shantsis** [comment]
> For the GIF issue, there's interesting tips here: https://css-tricks.com/gifs-and-prefers-reduced-motion/
> 
> To demonstrate the motion without the gif, you can just have two images side by side ([like I did here](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/):
> 
> image
> 
> We'll probably need something like this for the ebook anyway.

I like it! Long as we give that guidance to the authors it shouldn't be too hard to do. And good point for the ebook.

**Comment by dwsmart** [comment]
> > For the GIF issue, there's interesting tips here: https://css-tricks.com/gifs-and-prefers-reduced-motion/
> > To demonstrate the motion without the gif, you can just have two images side by side ([like I did here](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/):
> > image
> > We'll probably need something like this for the ebook anyway.
> 
> I like it! Long as we give that guidance to the authors it shouldn't be too hard to do. And good point for the ebook.

Happy to do that, should we just have a side by side anyway? I liked the animation, but sounds like it may be an issue?

**Comment by tunetheweb** [comment]
We’ll figure out a way of making both work. Always interesting to think about these things and add the abilities to our toolkit for future years!

Could you come up with the side by side image though and add to this branch? The. Leave the rest to me.

**Comment by dwsmart** [comment]
Absolutely, will do this later! will the gif and the static need to be the same dimensions?

**Comment by tunetheweb** [comment]
Nah. As presume the side by side one will be wider.

**Comment by dwsmart** [comment]
added `desktop-cls-example-static.png` & `mobile-cls-example-static.png`

File: src/content/en/2021/mobile-web.md [hunk 2]
```diff
@@ -140,12 +141,16 @@ Fortunately there are a few other sources. Paul Calvano ran some analysis on the
 
 
 
-Weekend days show a greater proportion of Mobile traffic, with 66.06% on Saturdays, **66.54%** on Sundays, and a slight increase to **59.06%**. Globally, not every country have Monday - Friday work weeks, Sunday - Thursday is another common pattern, (source: [Wikipedia](https://en.wikipedia.org/wiki/Workweek_and_weekend))
+Weekend days show a greater proportion of mobile traffic, with 66.06% on Saturdays, **66.54%** on Sundays, and a slight increase to **59.06%**. Globally, not every country has Monday to Friday work weeks -  Sunday to Thursday is also <a hreflang="en" href="https://en.wikipedia.org/wiki/Workweek_and_weekend">another common pattern</a>.
+
+{# TODO (Authors) - I dont get the use of bold here, and i'm confused what "a slight increase to" is referring to. A slight increase from? Is something missing? #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:147]: my bad, copied and pasted an older version over this bit at some point, re-written now

File: src/content/en/2021/mobile-web.md [hunk 3]
```diff
@@ -140,12 +141,16 @@ Fortunately there are a few other sources. Paul Calvano ran some analysis on the
...
 
 
 ##### Not all times are equal
 
-On weekdays, mobile usage dips vs. desktop usage around 5 am UTC and starts climbing again at 7pm UTC (with a small bump around 10 / 11am). This aligns with working hours.
+On weekdays, mobile usage dips vs. desktop usage around 5 AM UTC and starts climbing again at 7 PM UTC (with a small bump around 10 / 11 AM). This aligns with working hours.
+
+{# TODO (Authors) - I dont get what "mobile usage dips vs. desktop usage" is trying to say. One decreases while another increases? #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:156]: rewritten in last commit

File: src/content/en/2021/mobile-web.md [hunk 4]
```diff
@@ -400,6 +416,8 @@ Of the top five mobile web technologies, adoption rates for three were higher on
 
 JavaScript library JQuery is the dominant library of the mobile web, present in 84.38%.  Google is the dominant provider, holding three of the top five spots.  The most
 
+{# TODO (Authors) - Unsure if this is a typo or there's a thought missing here #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:424]: was typo, fixed in latest commit

File: src/content/en/2021/mobile-web.md [hunk 5]
```diff
@@ -556,8 +571,9 @@ Technology adoption rates for the mobile web moved in step with desktop.  The mo
 <figcaption>{{ figure_link(caption="Technology with higher desktop adoption rates.", sheets_gid="1172584192", sql_file="most_used_tech_by_domain_rank.sql") }}</figcaption>
 </figure>
 
+{# TODO (Authors) - Column alignment looks really off on this chart, its hard to read the last 3 columns #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:582]: missing header, fixed in lastest commit

File: src/content/en/2021/mobile-web.md [hunk 6]
```diff
@@ -605,17 +621,17 @@ Mobile sites were more likely to adopt certain technologies, but with a smaller
 <figcaption>{{ figure_link(caption="Technology with higher mobile adoption rates.", sheets_gid="1172584192", sql_file="most_used_tech_by_domain_rank.sql") }}</figcaption>
 </figure>
 
-
+{# TODO (Authors) - Column alignment looks really off on this chart, its hard to read the last 3 columns #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:634]: missing header, fixed in latest commit

File: src/content/en/2021/mobile-web.md [hunk 7]
```diff
@@ -629,7 +645,9 @@ The web is built on links.  On the mobile web, Unique Resource Identifier scheme
 
 The most prevalent URI schemes were https, found on 93.15% of sites, and it's non-secure equivalent, http, appearing on 56.65%.  The high use of non-secure link protocols is noteworthy as 2020 saw major announcements from browsers to protect users' safety by alerting them when content is not secure.
 
-After webpage links, the next five most used protocols in  anchor href values  on the mobile web are as follows:
+{# TODO (Authors) - I think this is a good spot to put an external link to read more on it, even if there's one example #}
```
[the remark this answers is not in the document]
Comment (dwsmart) [reply src/content/en/2021/mobile-web.md:659]: Added a link to `tel:` in latest commit

**Comment by dwsmart** [comment]
I addressed some of the great, valid points @shantsis, in the last commit I added a `{# TODO (Editor) - … #}` comment under yours for clarity, hope that's not breaking anything.

**Comment by tunetheweb** [comment]
Fixed the Gif thing. Can't get it to stop after two run throughs but do handle `prefers-reduced-motion` and ebook.

As mentioned in the chat:

I’ve just made another run through, more on the technical and “Web Almanac” consistency side and made quite a few edits (hope you don’t mind - but let me know if you massively disagree with any of them!).

Unfortunately, I’ve raised a few more TODOs that I’d like you to look at. Sorry about that! Have a search for them, and feel free to remove them when addressed, rather than adding another “Editor: done” line. I trust cha!

I’ve staged the latest version here: https://20211119t010018-dot-webalmanac.uk.r.appspot.com/en/2021/mobile-web

I’d like another run through of the graphs, and need to redo some of the screenshots after that, but it’s late here now so will leave that until tomorrow or the weekend.

It’s definitely almost there. Great read!

**Comment by tunetheweb** [comment]
Made some more edits for decimal place consistency, to add a couple more charts, increased the figure descriptions and run another spelling and grammar check.

I think this is good to merge.

@fellowhuman1101 / @dwsmart are you finished or still planning on tweaking it further? If not let's get this merged and finished!!

**Comment by dwsmart** [comment]
Hey @tunetheweb, we're good here too, and happy for it to be merged!

**Pull request merged**